buildbot/osuosl/master/config/builders.py

```diff
@@ -2019,6 +2019,38 @@
                         add_openmp_lit_args=["--time-tests", "--timeout 100"],
                     )},
 
+    {'name' : "openmp-offload-rhel-8.8",
+    'tags'  : ["openmp"],
+    'workernames' : ["rocm-worker-hw-04-rhel-8.8"],
```

Comment:
Unfortunately, a worker name must be an identifier, and this one is not. You need to come up with something not containing the dot symbol.

Reply:
Done